.github/workflows/unit-test.yml

```diff
@@ -165,7 +188,6 @@ jobs:
                   node --version
                   curl --version
                   git --version
-                  svn --version
```

Comment:
I've gone and removed this line since I noticed it while responding to a question above. SVN was sunset in GitHub Action runners earlier this year. I'm actually surprised this has not yet started producing failures. It did in Core.